src/hooks/forms/useSignInForm.ts

```diff
@@ -9,8 +9,8 @@ import { useAuth } from '../useAuth'
 import { SignInFormValues } from '~types/authForms'
 
 const defaultValues: SignInFormValues = {
-  email: 'test@example.com',
-  password: '123456',
+  email: '',
+  password: '',
```

Comment:
This values were left for debbuging and easier testing. You can do something like this here:

```suggestion
  // TODO: Reset this values when building production app
  email: 'test@example.com',
  password: '123456',
```